src/Data/ByteString/Base64.hs

```diff
@@ -55,7 +58,7 @@ encodeBase64' = encodeBase64_ base64Table
 -- use 'decodeBase64Unpadded'.
 --
 decodeBase64 :: ByteString -> Either Text ByteString
-decodeBase64 = decodeBase64_ NoPad decodeB64Table
+decodeBase64 = decodeBase64_ Padded decodeB64Table
```

Comment:
it used to be that 'NoPad' meant 'don't pad it out', which is confusing. Now, we have the adt that goes like this: 

- `Padded` = must be padded
- `Unpadded` = must be unpadded
- `Don'tCare` = don't care, and we'll pad out as needed.